test/test_node_handler.py

```diff
@@ -124,6 +124,7 @@ def test_create_node_endpoint(mock_get_current_user, mock_init_sub_id,
             'parent',
             'revision',
             'status',
+            'created',
```

Comment:
I also like to put the fields in alphabetical order, just to keep things easier to maintain. But that's not a requirement.